jumpscale/packages/vdc_dashboard/frontend/components/solutions/Download.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <base-dialog title="Download kubeconfig file" v-model="dialog">
```

Comment:
Should this component be more generic not for kube config only?

Reply:
indeed, that can be done for later ?